### PR TITLE
check on linters

### PR DESCRIPTION
**Pull request by leonard33**
checking linters pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (VICTOR-CHUKWU) [inline README.md:65]: - [ ] Your README should contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design], Kindly add that

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+
+
+ This is a Microverse consert website project.
+ 
+## Built With
+
+- HTML
+- css
+- javascript
+## Live Demo (not available)
```
Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] Kindly add a live website for this project hosted with github

Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] There should be a  video of a maximum length of  5 minutes. describing the features of your project, Kindly add that

File: about.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            </div>      
+        </section>
+        <section id="partners">
+            <div class="patner">
+                <img src="./assets/kbc.png" alt="kenya broadcasting"/>
+                <img src="./assets/kaka2.png" alt="kaka empire"/>
+                <img src="./assets/magik.png" alt="magic kenya"/>
+                <img src="./assets/mcsk2.png" alt="MCSK Kenya"/>
+                <img src="./assets/youth2.png" alt="youth kenya"/>
+            </div>
```
Comment (VICTOR-CHUKWU) [inline about.html:98]: - [ ] Kindly align these items to the center as it is on your home page to give your project uniformity
     👇 
          
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @leonard33 , 

## STATUS: REQUIRED CHANGES

✔️  You did a wonderful job on this project 👏 

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by leonard33** [comment]
-Acknowledging Cindy Shin
- webpage deploy link added in readme file (live demo view)
- Live demo video link describing the project added in readme file
- Alligned partners section inline done

**Comment by leonard33** [comment]
- Acknowledging Cindy Shin
- webpage deploy link added in readme file (live demo view)
- Live demo video link describing the project added in readme file
- Alligned partners section inline done

File: index.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                                        <img src="./assets/Vector.png" alt=""/>
+                                      <h6 class="paragraph">They sing sweet soulmusic in kiswahili very soothing to be presice.</h6>
+                                    </div>
+                                 </div>
+                            </div>
+                       </div>
+                       <!--modal stop-->
+                </div>
+            </div>
+            <button class="read-more">MORE<i class="fa fa-angle-down"></i></button>
```
Comment (gemmen29) [inline index.html:118]: [OPTIONAL] Kindly make the speakers when the more button pressed appear above it not below it for a better UI experience
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        </section>
+        <section id="footer-section">
+            <div class="footer">
+            <img src="./assets/wing2.png" alt=""/>
+            <h4>2022 Ultra Music Festival @copyright all rights reserved</h4>
+          </div>
+        </section>
```
Comment (gemmen29) [inline about.html:105]: - [OPTIONAL] Kindly change the `section` tag to the `footer` tag for semantic purposes.
- [OPTIONAL] Kindly change the footer to be dark in the desktop version to match the Behance design.
image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+                        <h4>Battle Of Band In Nairobi</h4>
+                    </div>
+                </div>
+                 <div class="past-holder2">
+                   <div class="overlay">
+                        <h3>2021</h3>
+                        <h4>Battle Of Band In Nairobi</h4>
+                    </div>
+                </div>   
+            </div>
```
Comment (gemmen29) [inline about.html:89]: - [OPTIONAL] Kindly change the text color to white and increase the ` font-size ` a little bit in this part to match the Behance design. 

| Your design | Behance design |
| --- | --- |
| image | image |

**Review by gemmen29 — APPROVED** [review]
Hi @leonard33, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**